apps/src/storage/dataBrowser/EditTableListRow.jsx

```diff
@@ -19,7 +20,19 @@ class EditTableListRow extends React.Component {
   };
 
   handleDelete = () => {
-    FirebaseStorage.deleteTable(this.props.tableName, this.props.tableType);
+    // TODO: post-firebase-cleanup, remove this conditional: #56994
+    if (isFirebaseStorage()) {
+      storageBackend().deleteTable(
+        this.props.tableName,
+        this.props.tableType,
+        refreshCurrentDataView
+      );
+    } else {
+      storageBackend().deleteTable(
+        this.props.tableName,
+        refreshCurrentDataView
+      );
+    }
```

Comment:
As before, `tableType` is detected automatically by the datablock storage backend, and doesn't need to be passed around on the frontend.